api-reference/subscribers/bulk-create-subscribers.mdx

```diff
@@ -2,4 +2,57 @@
 openapi: post /v1/subscribers/bulk
 ---
 
-<Snippet file="apikey-warning.mdx" />
\ No newline at end of file
+<Snippet file="apikey-warning.mdx" />
+
+
+<RequestExample>
+
+```javascript Node.js
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<NOVU_API_KEY>');
+
+await novu.subscribers.identify([
+  {
+    subscriberId: "1"
+    firstName: "Pawan";
+    lastName: "Jain";
+    email: "pawan.jain@domain.com";
+    phone: "+1234567890";
+    avatar: "https://gravatar.com/avatar/553b157d82ac2880237566d5a644e5fe?s=400&d=robohash&r=x";
+    locale: "en-US";
+    data: {
+      isDeveloper : true
+      customKey: "customValue"
+    };
+  },
+  {
+    subscriberId: "2"
+    firstName: "John";
+    lastName: "Doe";
+    email: "john.doe@domain.com";
+    phone: "+1234567891";
+    avatar: "https://gravatar.com/avatar/553b157d82ac2880237566d5a644e5fe?s=400&d=robohash&r=x";
+    locale: "en-UK";
+    data: {
+      isDeveloper : false
+      customKey1: "customValue1"
+    };
+  }
+  // more subscribers ...
```

Comment:
Hey @sumitsaurabh927 this is not correct. We use `identify` to create one subscriber which you should already be aware of by now because you have used it countless times while working on the quickstarts and docs.

This is about the bulk creation of subscribers. The correct method to use here is `bulkCreate`. I referenced the Node.js SDK as a place to check to see the functionality to bulk create. 
Did you not check this? https://github.com/novuhq/novu/blob/next/packages/node/src/lib/subscribers/subscribers.ts#L32

I also mentioned it in the v0.19 release docs https://dev.to/novu/whats-new-in-novu-019-1pcc

As you can see 
![Screenshot 2023-09-22 at 06 08 47](https://github.com/novuhq/docs/assets/2946769/a5697cd2-bb04-4df3-9bd7-0088a2f69742)

Reply:
Indeed I have and I thought I was missing something as using `identify` to create subscribers stood out to me. But I checked the node package and it was mentioned [there](https://github.com/novuhq/novu/tree/next/packages/node#bulk-create-subscribers) so I thought maybe I'm missing something here.